docs/pages/access-controls/guides/role-templates.mdx

```diff
@@ -367,14 +367,14 @@ Available interpolation functions include:
 
 Function | Description
 --- | ---
-`email.local(variable)` | Extracts the local part of an email field, like `Alice <alice@example.com>` or `bob@example.com`.
-`regexp.replace(variable, expression, replacement)` | Finds all matches of `expression` and replaces them with `replacement`. This supports expansion, e.g. `regexp.replace(external.email, "^(.*)@example.com$", "$1")`. Values which do not match the expression will be filtered out.
+`email.local(variable)` | Extracts the local part of an email address, `email.local(alice@example.com)` evaluates to `alice`.
+`regexp.replace(variable, expression, replacement)` | Finds all matches of `expression` and replaces them with `replacement`. This supports expansion, e.g. `regexp.replace(external.email, "^(.*)@example.com$", "$1")`. Values which do not match the expression will be filtered out. `$N` is used to refer to the Nth captured group.
```

Comment:
Might want to make it explicit that `$1` refers to the first captured group (rather than `$0`).